src/scss/includes/direction.scss

```diff
@@ -1,6 +1,32 @@
-.direction_panel {
-  background: url("../images/panel_top.svg") #f4f6fa repeat-x !important;
-  background-size: 100% 5px;
+.direction-panel {
+  background-color: $background;
+  @include card_shadow();
+  width: 400px;
+
+  position: absolute;
+
+  @media (min-width: 641px) {
+    @include panel_radius();
+  }
+
+  @media (max-width: 640px) {
+    position: fixed;
+    min-height: 42px;
```

Comment:
I don't think this is needed (but maybe I'm wrong, the style is tricky during the input focus).